specification/recoveryservicesbackup/resource-manager/readme.md

```diff
@@ -28,18 +28,21 @@ These are the global settings for the RecoveryServicesBackup API.
 title: Recovery Services Backup Client
 description: Open API 2.0 Specs for Azure RecoveryServices Backup service
 openapi-type: arm
-tag: package-2016-12
+tag: package-2017-07
+output-folder: ./
+csharp-sdks-folder: ./Generated/CSharp
 
 license-header: MICROSOFT_MIT
 ```
 
 
-### Tag: package-2016-12
+### Tag: package-2017-07
 
-These settings apply only when `--tag=package-2016-12` is specified on the command line.
+These settings apply only when `--tag=package-2017-07` is specified on the command line.
 
-``` yaml $(tag) == 'package-2016-12'
+``` yaml $(tag) == 'package-2017-07'
 input-file:
+- Microsoft.RecoveryServices/2017-07-01/jobs.json
 - Microsoft.RecoveryServices/2016-12-01/backupManagement.json

```

Comment:
Didn't understand this point either. Swagger is not deleted, only api version for 2 operations has been updated. Those two operations have been moved to jobs.json. 
So as a whole number of operations are constant, just 2 operations have new api version.